platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -301,6 +291,18 @@ demonstrate their direct impact on and relationships with product and value
 stream teams (see the previous two paragraphs), presenting the platform team as
 a strategic partner of product teams in delivering value to customers.
 
+## Enabling platform teams
```

Comment:
@abangser what do you think of this?
I'm not quite happy with the wrap up here. The challenges section talks about productisation, capabilities, and organisational support and also offers some solutions.
Now we get to a section named "enabling platform teams" and it only talks about reducing load, but doesn't refer to any prior established point. It feels like something's missing.

Does that make sense?

Reply:
That does make sense. What I hear (and agree with) is that this section feels like a return to the beginning of challenges rather than a nice wrap up that leaves the reader understanding challenges but also given opportunities to succeed. I think this is a byproduct of it previously being above the challenges section.

I started a suggestion below, but decided that keeping the conversation in this thread would be easier to follow. So please see this as a suggestion as a replacement for the paragraph in this section which tries to introduce the idea that there are support systems to the challenges that we have presented.

Please let me know if this helps with the concerns you have 🤞 

> It is clear from these challenges that platform teams are faced with a number of 
> diverse challenges that lead to cognitive load. Just as with their application team
> counterparts, this challenge grows with the number and diversity of users and 
> teams they need to support.
> 
> It is important to focus the platform team's energy on the experience and 
> capabilities that are unique to their specific business. Ways to reduce load on 
> the platform team include the following:
> 
> 1. Use implementations from managed service providers where reasonable
> 1. Leverage open source frameworks and toolkits for creating docs, templates and
>    compositions for application team use
> 1. Ensure platform teams are staffed appropriately for their domain and number
>    of customers

Reply:
Yes, this reads much better in the context of the above section. I like it! :)